src/App/ComponentsDocumentation/components/Alerts/constants.js

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import Alert from "@components/Alert";
+
+const exampleText = <p><b>Error title:</b> Input informative alert message here, provide link to solution or more info when <a href="#">possible</a></p>;
+
+export const alertsShowCase = {
+    id: "overviewAlert",
+    tabsId: "overviewAlertTabs",
+    elements: [
+        {
+            tab: "Error",
+            component: <Alert type="danger" icon="cancel" text={exampleText} />,
+            options: {
+                checkbox: [
+                    {
+                        title: "Add",
```

Comment:
Please change to "Options" for keeping consistency. 💯 
Great work daddy cool!

Reply:
😂